### PR TITLE
refactor(saves): replace my list with saves [IN-736]

### PR DESCRIPTION
**Pull request by timc-mozilla**
## Summary
* replacing my list with saves

## References 
* [IN-736](https://getpocket.atlassian.net/jira/software/projects/IN/boards/80?selectedIssue=IN-736)

## Implementation Details
* refactored `My list` and variations of it with `saves`

## Test Steps
* Write out what QA should do to verify this change works as expected and hasn't introduced regressions. Can mention specific OS versions, devices, areas of the app to test as needed.

## PR Checklist:
- [x] Added Unit / UI tests
- [x] Self Review (review, clean up, documentation, run tests)
- [x] Basic Self QA

### CODE REVIEW

**Comment by timc-mozilla** [comment]
The failure is due to a fix made in this PR https://github.com/Pocket/pocket-ios/pull/289

seems like its happening more often than not.

File: Tests iOS/Support/Elements/PocketAppElement.swift [hunk 1]
```diff
@@ -27,8 +27,8 @@ struct PocketAppElement {
         return HomeViewElement(app.otherElements["home"])
     }
 
-    var myListView: MyListElement {
-        return MyListElement(app)
+    var mySaves: SavesElement {
```
Comment (dskuza) [inline Tests iOS/Support/Elements/PocketAppElement.swift:30]:
```suggestion
    var saves: SavesElement {
```

What do you think?

**Review by dskuza — APPROVED** [review]

**Comment by dskuza** [comment]
I appreciate the renames of actual code-level things and not just strings.

File: PocketKit/Sources/Analytics/Context/UIContext.swift [hunk 2]
```diff
@@ -64,7 +64,7 @@ extension UIContext {
 extension UIContext {
     public enum Identifier: String, Encodable {
         case home
-        case myList = "my_list"
+        case saves = "saves"
```
Comment (cyndichin) [inline PocketKit/Sources/Analytics/Context/UIContext.swift:67]: should we change this? this might change analytics

**Pull request merged**